app/studentsApi.js

```diff
@@ -0,0 +1,45 @@
+const express = require("express")
+const Router = express.Router()
+const studentDB = require("./studentsDB.json")
+
+//api
+
+function catchStudent(studentId){
```

Comment:
O nome da função com o início catch não é condizente com este projeto. Está claro o CTRL+C e CTRL+V do projeto desenvolvido no laboratório.

Tenho certeza que você pode fazer melhor @simaofaustino123 !